lib/guardian_db.ex

```diff
@@ -81,26 +82,43 @@ defmodule GuardianDb do
   end
 
   @doc """
-  After the JWT is generated, stores the various fields of it in the DB for tracking
+  After the JWT is generated, stores the various fields of it in the DB for tracking if
+  token type matches the configured types to be stored.
   """
   def after_encode_and_sign(resource, type, claims, jwt) do
-    case Token.create!(claims, jwt) do
+    case store_token(type, claims, jwt) do
```

Comment:
With the changes here...it looks like if `store_token` returns `:non_storable_type` it will still end up returning the `:ok` tuple variant.

Reply:
If the token type should not be stored, the claims should just pass through `after_encode_and_sign/4`. 
This is why `store_token` is intentionally returning an `:ok` variant for these token types. An `:error` tuple would only return if there was a problem with `Token.create!/2`.

Is there a clearer way to write this or should I update the fn docs to convey what this is doing?